README.md

```diff
@@ -5,9 +5,14 @@
 </p>
 
 ---
-![example workflow](https://github.com/slundberg/shap/actions/workflows/run_tests.yml/badge.svg)
+[![PyPI](https://img.shields.io/pypi/v/shap)](https://pypi.org/project/shap/)
```

Comment:
Shall we also add a badge for the supported python versions?

```
[![PyPI pyversions](https://img.shields.io/pypi/pyversions/shap)](https://pypi.org/pypi/shap/)
```

[![PyPI pyversions](https://img.shields.io/pypi/pyversions/shap)](https://pypi.org/pypi/shap/)

Reply:
Good idea!